src/hip/handlehip.cpp

```diff
@@ -447,6 +447,13 @@ void Handle::AddProgram(Program prog,
     this->impl->cache.AddProgram(prog, program_name, params);
 }
 
+void Handle::ClearProgram(const std::string& program_name, const std::string& params) const
+{
+    this->impl->cache.ClearProgram(program_name, params);
+}
+
+void Handle::ClearProgram() const { this->impl->cache.ClearProgram(); }
```

Comment:
This should probably be named `ClearPrograms` to make it clearer that it removes all programs.